templates/base/head_script.tmpl

```diff
@@ -41,6 +41,12 @@ If you introduce mistakes in it, Gitea JavaScript code wouldn't run correctly.
 			copy_error: '{{.locale.Tr "copy_error"}}',
 			error_occurred: '{{.locale.Tr "error.occurred"}}',
 			network_error: '{{.locale.Tr "error.network_error"}}',
+			contributions: '{{.locale.Tr "contributions"}}',
```

Comment:
Hmm, I don't know if I prefer this unconditional overhead.
After all, these things will be sent on every request.
Perhaps we should do something similar to what I did in https://github.com/go-gitea/gitea/blob/29c00eb1ed13fbcd2a5e493d075c2bbd0e2a26f7/templates/repo/diff/box.tmpl#L30-L31  combined with https://github.com/go-gitea/gitea/blob/29c00eb1ed13fbcd2a5e493d075c2bbd0e2a26f7/web_src/js/features/pull-view-file.js#L15-L17:
Set the text as an attribute of the corresponding element, and then update the text within JS.

Reply:
Done